src/chigame/forums/views.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.messages.views import SuccessMessageMixin
+from django.views.generic import CreateView
+from machina.apps.forum.models import Forum
+
+
+class ForumCreateView(LoginRequiredMixin, SuccessMessageMixin, CreateView):
+    model = Forum
+
+    success_message = "Forum successfully created!"
+    template_name = "forum_form.html"
+    fields = ["name", "slug", "description", "image", "type"]
```

Comment:
Is the slug here something based on the machina implementation and okay to be something inputted from the user? I imagine that the slug could be an autoincrementing attribute (or alike) rather than something provided from the user.

Reply:
The slug is from the machina implementation, and looking at their model doesn't have a `blank=null` field in the model, so I assumed they'd want that field to be non-optional on forms. Remember, the user here will always be an admin, so you can imagine they'd likely have some idea of how they'd want the url scheming/naming to work and thus customization over those urls.

Not fully sold on exposing it as a field, so let me know what you think!

Reply:
I think for the most part, if it's in the machina implementation, it doesn't cause any functional issues. We can leave as is and open an issue if we find that it is not necessary or we need a different way of employing slugs.